tests/x11/evolution/evolution_timezone_setup.pm

```diff
@@ -45,7 +45,7 @@ sub run {
     if (check_screen("timezone-asia")) {
         send_key("right");
         send_key_until_needlematch("timezone-yerevan", "end");
-        assert_and_click("timezone-shangai");
+        assert_and_click("timezone-yerevan");
```

Comment:
I would recommend to squash commits. There is no need to track typos/fixes during test development and introduce new commits in history.